scripts/data_migrations/2019-04-24_add-ethnicity-in-the-uk-redirect.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO redirect (created, from_uri, to_uri) VALUES (now(), 'ethnicity-in-the-uk', 'uk-population-by-ethnicity');
```

Comment:
We technically don't need to make this a data migration as we have a management command for adding redirects. So I've just used that on my own PR #1070. But I guess either works 👍

Reply:
@samuelhwilliams ah yeah, I forgot about that. Maybe this is a bit less error-prone?

Reply:
Probably true. In which case we should drop the management command.

Reply:
Maybe? One for discussion.